File: Python/1008.py
number = int(input())
workHoursInMonth = float(input())
salaryPerHour = float(input())

salary = (workHoursInMonth / 30) * salaryPerHour * 30

print("NUMBER = %i \nSALARY = U$ %.2f" % (number, salary))

File: C++/1008/1008.cpp
#include <iostream>
using namespace std;

int NUMBER, work_hour;
double SALARY, total;

int main() {
	scanf("%i", &NUMBER);
	scanf("%i", &work_hour);
	scanf("%lf", &SALARY);
	
	total = work_hour*SALARY;

	printf("NUMBER = %i\nSALARY = U$ %.2lf\n", NUMBER, total);
}
